plotter/image_test.go

```diff
@@ -53,3 +53,46 @@ func ExampleImage() {
 func TestImagePlot(t *testing.T) {
 	cmpimg.CheckPlot(ExampleImage, t, "image_plot.png")
 }
+
+// An example of embedding an image in a plot with non-linear axes.
+func ExampleImage_log() {
+	p, err := plot.New()
+	if err != nil {
+		log.Fatalf("error: %v\n", err)
+	}
+	p.Title.Text = "A Logo"
+
+	// load an image
+	f, err := os.Open("testdata/image_plot_input.png")
```

Comment:
Can we use a different input image here. The effect of the transform is not very clear. I would suggest something that is less right-angular.